src/Sulu/Bundle/PreviewBundle/Resources/config/config_preview.yml

```diff
@@ -3,3 +3,6 @@ framework:
         storage_id: session.storage.mock_file
     profiler:
         enabled: false
+
+sulu_document_manager:
```

Comment:
@danrot @chirimoya looks good to me. we should think about moving a basic ´WebsiteKernel` to sulu-core which would avoid coupling the `PreviewKernel` to the application.